conjure: This should never happen, sorry!

However, it did happen, so it must be a bug. Please report it to us!

Conjure is actively maintained, we will get back to you as soon as possible.
You can help us by providing a minimal failing example.

Also include the repository version for this build: 3471a2936 (2019-05-08 11:52:07 +0100)

Issue tracker: http://github.com/conjure-cp/conjure/issues



Not refined: s_ExplicitR20
Domain     : matrix indexed by [int(1..fin1)] of permutation {PermutationAsFunction} of int(1..3)
    Context #1: s_ExplicitR20[q3]
    Context #2: image(s_ExplicitR20[q3], sequence(1, 2, 3))
    Context #3: image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm
    Context #4: [image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm | q3 : int(1..fin1)]
    Context #5: and([image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm | q3 : int(1..fin1)])
Not refined: sequence(1, 2, 3)
    Context #1: image(s_ExplicitR20[q3], sequence(1, 2, 3))
    Context #2: image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm
    Context #3: [image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm | q3 : int(1..fin1)]
    Context #4: and([image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm | q3 : int(1..fin1)])
Not refined: superperm
Domain     : sequence {ExplicitBounded} (maxSize 100) of int(1..3)
    Context #1: image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm
    Context #2: [image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm | q3 : int(1..fin1)]
    Context #3: and([image(s_ExplicitR20[q3], sequence(1, 2, 3)) substring superperm | q3 : int(1..fin1)])

CallStack (from HasCallStack):
  error, called at src/Conjure/Bug.hs:21:15 in conjure-cp-2.2.0-71NFmy1yFuTKzqIQD5mEC7:Conjure.Bug
  bug, called at src/Conjure/UI/Model.hs:936:26 in conjure-cp-2.2.0-71NFmy1yFuTKzqIQD5mEC7:Conjure.UI.Model
cat: conjure-output/*.solution: No such file or directory
